streaming/trade-generator/src/main/java/com/hazelcast/jet/sample/tradegenerator/GenerateTradesP.java

```diff
@@ -105,12 +108,27 @@ public boolean complete() {
         long timestamp = currentTimeMillis();
         long count = 0;
         for (; nextSchedule <= now; nextSchedule += periodNanos, count++) {
-            String ticker = tickers[rnd.nextInt(tickers.length)];
+            String ticker = tickers.get(rnd.nextInt(tickers.size()));
             emit(new Trade(
                     timestamp - rnd.nextLong(MAX_LAG),
-                    ticker, QUANTITY, tickerToPrice.get(ticker)));
+                    ticker, QUANTITY, rnd.nextInt(5000)));
         }
         TOTAL_EVENT_COUNT.addAndGet(count);
         return false;
     }
+
+    public static void loadTickers(JetInstance jet) {
+        try (BufferedReader reader = new BufferedReader(new InputStreamReader(
+                GenerateTradesP.class.getResourceAsStream("/nasdaqlisted.txt"), StandardCharsets.UTF_8))
+        ) {
+            Map<String, Integer> tickers = jet.getMap(TICKER_MAP_NAME);
+            reader.lines()
+                  .skip(1)
+                  .limit(100)
```

Comment:
Now that this code is generalized and extracted to a separate module, this magic number should become a parameter.